config/defaults.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  taskcluster: {
+    queueBaseUrl: 'https://queue.taskcluster.net/v1',
+    authBaseUrl: 'https://auth.taskcluster.net/v1',
```

Comment:
Hmm... is there a good reason not to use the defaults from the taskcluster-client package. And just define them as `undefined` here.